server/src/test/java/io/crate/expression/operator/all/AllLikeOperatorTest.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ALL KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.operator.all;
+
+import static io.crate.testing.Asserts.isLiteral;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.Test;
+
+import io.crate.expression.scalar.ScalarTestCase;
+
+public class AllLikeOperatorTest extends ScalarTestCase {
+
+    @Test
+    public void testNormalizeSingleSymbolEqual() {
```

Comment:
the name is not matching the tests, maybe `Simple` is better, or maybe merge all this tests in `testEvaluateStraight()` ?

Reply:
Thanks, changed to `Simple` since `testEvaluateStraight` evaluates while `testNormalizeSingleSymbolEqual` normalizes.